binary-install/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "binary-install"
+version = "0.1.0"
+
+[dependencies]
+curl = "0.4.13"
+dirs = "1.0.4"
+failure = "0.1.2"
+flate2 = "1.0.2"
+hex = "0.3"
+siphasher = "0.2.3"
+slog = "2.3"
+slog-term = "2.4"
+slog-async = "2.3"
```

Comment:
I think not all of these dependencies (like this one) may be needed here, mind pruning this list of those which aren't needed?

Reply:
I will remove them.